test/aave-v2/TestSupplyVault.t.sol

```diff
@@ -140,4 +140,30 @@ contract TestSupplyVault is TestSetupVaults {
         vm.expectRevert("ERC4626: redeem more than max");
         vaultSupplier1.redeemVault(daiSupplyVault, shares + 1);
     }
+
+    function testNotOwnerShouldNotTransferTokens(
+        address _caller,
```

Comment:
Does caller really need to be fuzzed here?

Reply:
Yes we can remove it, good point.